runtime/queries/bass/highlights.scm

```diff
@@ -89,13 +72,60 @@
   (_)
   (symbol) @function)
 
-;;; generic highlighting for all forms
+;; Functions
+
+(list
+  . (symbol) @function)
 
-; first symbol in a list form is a combiner call
-(list . (symbol) @function)
+;; Variables
+
+(list (symbol) @variable)
 
-; highlight symbols as vars only when they're clearly vars
 (cons (symbol) @variable)
+
 (scope (symbol) @variable)
-(path form: (symbol) @variable)
-(symbind form: (symbol) @variable)
+
+(symbind (symbol) @variable)
+
+;; Namespaces
+
+(symbind
+  (symbol) @namespace
+  . (keyword))
+
+;; Punctuation
+
+[ "(" ")" ] @punctuation.bracket
+
+[ "{" "}" ] @punctuation.bracket
+
+[ "[" "]" ] @punctuation.bracket
+
+((symbol) @punctuation.delimiter
+  (#eq? @punctuation.delimiter "->"))
+
+;; Literals
+
+(string) @string
+
+(escape_sequence) @constant.character.escape
+
+(path) @string.special.path
+(command) @string.special.path
+
+(number) @constant.numeric.integer
```

Comment:
Does `number` capture integers and floats? `@constant.numeric` would be more appropriate if it is both but the current capture is good if it's just integers